runbot/runbot.py

```diff
@@ -1555,7 +1568,7 @@ def badge(self, repo_id, branch, theme='default'):
         domain = [('repo_id', '=', repo_id),
                   ('branch_id.branch_name', '=', branch),
                   ('branch_id.sticky', '=', True),
-                  ('state', 'in', ['testing', 'running', 'done']),
+                  ('state', 'in', ['testing', 'running', 'done']), # don't know what this route does, need to add 'deathrow' ?
```

Comment:
This route return the image for the badge shown in README of odoo/odoo [1]
You need to ignore build manually killed (next line)

[1] https://github.com/odoo/odoo/#readme